trusted-firmware-m/platform/ext/common/tfm_platform.c

```diff
@@ -13,7 +13,11 @@
 __WEAK enum tfm_hal_status_t tfm_hal_platform_init(void)
 {
     __enable_irq();
+
```

Comment:
Nothing wrong in this line, but please refer to the upstream PR in the commit message.

And note, this fork is a fork from here:
https://github.com/zephyrproject-rtos/trusted-firmware-m/

not here:
https://review.trustedfirmware.org/plugins/gitiles/TF-M/trusted-firmware-m/+/refs/heads/master

so please refer to the correct upstream PR to make life easier during upmerges.

Reply:
I am a bit confused, what do you mean the correct PR? 
Should I open a new PR here:
https://github.com/zephyrproject-rtos/trusted-firmware-m/

Never did that until now I think.

Reply:
nevermind, seems like in our tf-m fork the most common thing is to refer to the up-upstream repo, and not just to the first upstream repo (Zephyr's fork)